src/Api/Controller/TokenController.php

```diff
@@ -67,8 +75,9 @@ public function createAction(Request $request) : JsonResponse
             return new JsonResponse(['errors' => $e->getErrors()], 400);
         }
 
-        /** @var Payment $payment */
-        if (false == $payment = $this->payum->getStorage(Payment::class)->find($data['paymentId'])) {
+        $payment = $this->paymentStorage->findById($data['paymentId']);
```

Comment:
not sure `findById` method exist on PaymentStorage class.

Reply:
It is.